### ISSUE TITLE
Detroit - include % rent options 

### ISSUE THREAD

**Issue by kathyccheng** [CityOfDetroit/bloom#1292]
As a resident
I might want to include % rent options in the rent filter 

Acceptance Criteria 
* When I select rent ranges for filtering, it should automatically include % income amounts 

**Mockup**
https://www.figma.com/file/D2fNPS5Ucll8uz7hyMUQGB/bloom-public-single-listing?node-id=1285%3A19800

something to note that acceptance criteria on this ticketo has changed a bit:

we just need the filter to include listings based on income even when there’s a flat rent value range indicated. we just won’t need the checkbox. so the UI would look the same but the filter would update to include the listings based on income

is the new criteria

**QA NOTES**
_on the partner site_
edit or add a listing such that it would show on the public site, and has at least 1 unit group with a percent of income value for rent on there

_on the public site_
head to the listing grid
verify the listing you edited/created is present
go to add a min and/or max rent filter 
verify your listing shows even if its flat rent values may not indicate that it should show

basically as long as a listing has a percent of income value set, then the listing should show regardless of the min/max rent filters

**Comment by kathyccheng** [comment]
@slowbot fyi Safiya wants the % based ones automatically included in the filtering so updated the issue to reflect that

**Comment by kathyccheng** [comment]
LGTM
